### DISCUSSION TITLE
DAI 2023-2024 - My IntelliJ + Maven project - Sarah Jallon

### DISCUSSION THREAD

**Discussion by sarsurgithub** [heig-vd-dai-course/heig-vd-dai-course#80] [Show and tell]
https://github.com/sarsurgithub/java-intelliji-idea-and-maven

the branch is master and not main, even if I used the command u specified when initing git, here is what it told me even if the folder did not already use git and is not in a folder that is using git
 ```
git init --initial-branch=main
warning: re-init: ignored --initial-branch=main
Reinitialized existing Git repository in C:/Users/emili/Documents/HEIG-VD/DAI/java-intelliji-idea-and-maven/.git/
```

**Comment by ludelafo** [comment]
My guess is that IntelliJ IDEA did initialize the Git repository for you, using its default `master` branch. The directory `.git` is hidden by default from your IDE and OS. From my understanding, the `.git` was indeed present (from the message `Reinitialized existing Git repository`). If you try the same command in a new directory, it should work.


**Edit**: You could add a few lines on what is the project about, how to build and run the application using Maven, etc. to have a project that could be understood by all people checking your project. :)

**Reply by sarsurgithub** [reply]
Makes sense! I thought about that as well but found it weird as I didn't remember clicking on anything git related in my idea. Thanks for your answer

**Reply by ludelafo** [reply]
You're welcome!
